docs/codeql/codeql-cli/getting-started-with-the-codeql-cli.rst

```diff
@@ -129,12 +129,12 @@ see ":doc:`About QL packs <about-ql-packs>`."
      ":doc:`Upgrading CodeQL databases <upgrading-codeql-databases>`."
     
    - For the queries used on `LGTM.com <https://lgtm.com>`__, check out the
```

Comment:
Since this is the most common use case, maybe we can put this bulletpoint at the top?

Reply:
I thought about that but wasn't confident in my understanding. Thanks, will do! ⚡